dashboard/dashboard.html

```diff
@@ -328,5 +335,7 @@ <h2>Amherst COVID Data Dashboard</h2>
 
     </script>
 
+    <p>Notes on Data: Data is Amherst only. Data for weekends is not always available. Missing dates are filled in with prior know date. This data was last updated on Feb 26, 2021. <a href="https://docs.google.com/spreadsheets/d/16YiakYsj_PkZjN6PSnY-6xhUDBjEH4hWkcQQPIcxV3w">View Data Table</a></p>
```

Comment:
'know' typo
I kind of like the .csv solution better as it fully removes the view from the backend, but let's have that for v2, it's a bit more painful to implement